lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -586,12 +586,12 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
             cohorts_group_name = []
             if course.is_cohorted:
                 group = get_cohort(student, course_id, assign=False)
-                cohorts_group_name.append(group.name if group else '')
+                cohorts_group_name.append(unicode(group.name) if group else u'')
 
             group_configs_group_names = []
             for partition in experiment_partitions:
                 group = LmsPartitionService(student, course_id).get_group(partition, assign=False)
-                group_configs_group_names.append(group.name if group else '')
+                group_configs_group_names.append(unicode(group.name) if group else '')
```

Comment:
@cpennington - what about here? Groups are a namedtuple (stored on course.user_partitions.groups). Will the modulestore return their names as unicode or ascii-encoded strings?